game.py

```diff
@@ -0,0 +1,90 @@
+"""
+Main Runner for WTT game
+"""
+import datetime
+import sqlite3
+from ascii_img.ask_user_name import ask_name
+from ascii_img.credential import credential
+from ascii_img.help import rules
+from ascii_img.intro import intro
+from scripts.models import Enemy
+from scripts.models import Player
+
+
+def play():
+    """
+    main function for game and write data to db
+    """
+    ask_name()
+    player_name = input("   ")
+    player = Player(player_name)
+    level = 1
+    enemy = Enemy(level)
+    while player.lives > 0:
+        player.attack(enemy)
+        if enemy.level == 0:
+            level += 1
+            enemy = Enemy(level)
+            player.score += 5
+        print(f"Your score is {player.score}")
+        player.defence(enemy)
```

Comment:
If enemy down. The new enemy will attack. This can make some not obvious behavoir